stories/banner.stories.js

```diff
@@ -0,0 +1,263 @@
+            </g>
+          </svg>
+        </div>
+        <p>Uh oh, something is wrong. Please review the wrong thing.</p>
+      </div>
+    </Demo>
+    <Description>
+      <h1>Error</h1>
+      <p>Use to surface an error to the user, usually as part of a form.</p>
+    </Description>
+    <Code
+      tabs={{
+        html: `<!-- banner -->
+<div role="alert" className="Manifold__Banner Manifold__Banner--Error">
+  <div className="Manifold__Banner__Icon">
+    <svg
+      aria-label="alert-triangle"
+      role="img"
+      viewBox="0 0 24 24"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <g fill-rule="evenodd" clip-rule="evenodd">
+        <path
+          d="M10.528 2.283a3 3 0 014.037 1.058l.003.005 8.47 14.14.008.014a3 3 0 01-2.565 4.5H3.519a2.999 2.999 0 01-2.565-4.5l.008-.014 8.47-14.14.858.514-.855-.519a3 3 0 011.093-1.058zm.618 2.094L2.683 18.506A1 1 0 003.536 20h16.928a1 1 0 00.853-1.494L12.855 4.379l-.001-.002a1 1 0 00-1.708 0z"
+        />
+        <path
+          d="M11 17a1 1 0 011-1h.01a1 1 0 110 2H12a1 1 0 01-1-1zM12 8a1 1 0 011 1v4a1 1 0 11-2 0V9a1 1 0 011-1z"
+        />
+      </g>
+    </svg>
+  </div>
+  Uh oh, something is wrong. Please review the wrong thing.
+</div>`,
+        scss: `${comment.block(title)}
+
+.Manifold__Banner {
+  @include mercury.Manifold__Banner;
+
+  ${comment.modifier}
+
+  &--Error {
+    @include mercury.Manifold__Banner--Error;
+  }
+
+  ${comment.element}
+
+  &__Icon {
+    @include mercury.Manifold__Banner__Icon;
+  }
+}`,
+      }}
+    />
+  </Story>
+);
+
+export const success = () => (
+  <Story>
+    <Demo>
+      <div role="alert" className="Manifold__Banner Manifold__Banner--Success">
+        <div className="Manifold__Banner__Icon">
+          <svg aria-label="check" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+            <path
+              fill-rule="evenodd"
+              clip-rule="evenodd"
+              d="M20.707 5.293a1 1 0 010 1.414l-11 11a1 1 0 01-1.414 0l-5-5a1 1 0 111.414-1.414L9 15.586 19.293 5.293a1 1 0 011.414 0z"
+            />
+          </svg>
+        </div>
+        <p>Look at that! An absolute success, nice work.</p>
+      </div>
+    </Demo>
+    <Description>
+      <h1>Success</h1>
+      <p>Use to surface a successful message to the user, usually as part of a form.</p>
+    </Description>
+    <Code
+      tabs={{
+        html: `<!-- banner -->
+<div role="alert" className="Manifold__Banner Manifold__Banner--Success">
+  <div className="Manifold__Banner__Icon">
+    <svg
+      aria-label="check"
+      role="img"
+      viewBox="0 0 24 24"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M20.707 5.293a1 1 0 010 1.414l-11 11a1 1 0 01-1.414 0l-5-5a1 1 0 111.414-1.414L9 15.586 19.293 5.293a1 1 0 011.414 0z"
+      />
+    </svg>
+  </div>
+  Look at that! An absolute success, nice work.
```

Comment:
Whatever we decide re: icons, the HTML should probably be updated (either with a `<p>` wrapper, or if we decide to go icon-in-CSS